build/components/setup.tex

```diff
@@ -93,50 +148,92 @@
     \fi
 }
 
-% COUNTER (Zweck: in römischen Zahlen weiterzählen, nachdem der Counter von arabisch zurückgeändert wird)
-\newcounter{savepage}
 
-% Sections sollen mit Seitenumbruch beginnen
-\let\stdsection\section
-\renewcommand\section{\newpage\stdsection}
+% ANLAGENVERZEICHNIS SETUP
+% definiert eine neue Liste für das Anlagenverzeichnis
+\newcommand{\listanlageverzeichnis}{\vspace*{-20pt}}
+\newlistof{anlagen}{alt}{\listanlageverzeichnis}
 
-% Überschreiben von \mathrm{} -> einheitlichen Abstand einfügen
-\let\oldMathrm\mathrm
-\renewcommand{\mathrm}[1]{\,\oldMathrm{#1}}
+% Befehl welcher ein Item dem Anlagenverzeichnis hinzufügt
+\newcommand{\addItemToAnlageverzeichnis}[1]{%
+    \def\fig{fig}
+    \def\tab{tab}
 
-% Template-Root-Verzeichnis ist das neue Arbeitsverzeichnis
-% muss nach \input{setup.tex} aufgerufen werden, damit alle Template-Root-Dateien integriert werden
-\makeatletter
-\def\input@path{{../}{path1/}}
-\makeatother
+    \ifx\fig\typeOfCap
+        \def\type{\thefigure}
+        \def\name{Abb.\hspace{8pt}}
+    \else \ifx\tab\typeOfCap
+            \def\type{\thetable}
+            \def\name{Tab.\hspace{10pt}}
+        \fi
+    \fi
+    \setcounter{anlagen}{\type}
 
-% die veraltete CAUTHOR Variable wird automatisch befüllt
-\def\CAUTHOR{\CAUTHORVOR\ \CAUTHORNACH}
+    % only here because the \type-counter is one lower (later it will count up like normal)
+    % -> reason... it's called too early but can't called later because of dependencies other types
+    % works only in the last section of the paper so it should be fine :)

```

Comment:
WE ARE A VERY DEUTSCHES REPOSITORY

Reply:
Falls jemand #123 bearbeitet sollte dies überarbeitet werden. Ist nur hier um aus `com.tex` in setup zu wandern.